src/components/SubjectInterest.vue

```diff
@@ -3,88 +3,148 @@
+          <br>
+        </div>
+
+        <span v-bind:class= "{dim: !(selectedOptionTwo === null || selectedOptionTwo === 'Two')}">
+          <input  type="radio" id="two" value="Two" v-model="selectedOptionTwo">
+          <label class="custom-input" for="two"> parts of this study</label></label>
+          <br>
+        </span>
 
-    <br>
-    <br>
-    <button v-focus="isEligible" :disabled="!isEligible" id="next" v-on:click="clicked"> Next </button>
+      </div>
 
+    </div>
+    <div class="row">
+    <button class="text-center" type="submit" v-if="isEligible" v-focus="isEligible" id="next" v-on:click="clicked"> Next </button>
+    </div>
   </div>
 </template>
 
 <script>
-  import { Focus } from '@/directives/focus.js'
-  export default {
-    name: 'subjectInterest',
-    data () {
-      return {
-        selectedOptionOne: '',
-        selectedOptionTwo: ''
-      }
+import { Focus } from '@/directives/focus.js'
+import _ from 'lodash'
+import Multiselect from 'vue-multiselect'
+
+export default {
+  name: 'subjectInterest',
+  data () {
+    return {
+      selectedOptionOne: null,
+      selectedOptionTwo: '',
+      isInterested: false,
+      isWilling: false,
+      isEligible: false,
+      value: [
+        { name: 'Vue.js' }
+      ],
+      options: [
+        { name: 'have parkinsons' },
+        { name: 'have another movement disorder' },
+        { name: 'would like to be a control subject' }
+      ]
+    }
+  },
+  watch: {
+    selectedOptionOne: function (newOption) {
+      this.setIsInterested()
     },
-    computed: {
-      isEligible: function () {
-        return (this.isInterested && this.isWilling)
-      },
-      isInterested: function () {
-        return (this.selectedOptionOne !== '')
-      },
-      isWilling: function () {
-        return (this.selectedOptionTwo !== '')
-      }
+    selectedOptionTwo: function (newOption) {
+      this.setIsWilling()
     },
-    methods: {
-      clicked () {
-        if (this.isEligible) {
-          this.$router.push('Eligibility')
-        }
+    isWilling: function (newValue) {
+      this.setIsEligible()
+    }
+  },
+  methods: {
+    clicked () {
+      if (this.isEligible) {
+        this.$router.push('Eligibility')
       }
     },
-    directives: {
-      Focus
-    }
+    setIsInterested: _.debounce(
+      function () {
+        this.isInterested = (this.selectedOptionOne.length > 0)
+      },
+      500),
+    setIsWilling: _.debounce(
+      function () {
+        this.isWilling = (this.isInterested && this.selectedOptionTwo !== '')
+      },
+      500),
+    setIsEligible: _.debounce(
+      function () {
+        this.isEligible = (this.isWilling && this.isInterested)
+      }, 250
+    )
+  },
+  directives: {
+    Focus
+  },
+  components: {
+    Multiselect
   }
+}
 </script>
 
+<style src="vue-multiselect/dist/vue-multiselect.min.css"></style>
 
 <style>
-  button:disabled {
-    opacity: 0.5;
-  }
+button:disabled {
```

Comment:
Some styles are repeated and should probably go in App.vue, e.g.
`button:disabled, .dim {
  opacity: 0.5;
}`